### PR TITLE
Add setting for project - update information

### PR DESCRIPTION
**Pull request by phanbt58**

### CODE REVIEW

File: app/views/projects/_setting_information.html.erb [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+        </div>
+      </div><br>
+
+      <div class="row">
+        <div class="col-md-2 text-right">
+          <span><%= f.label :start_date %></span>
+        </div>
+        <div class="col-md-3">
+          <%= f.text_field :start_date, class: "form-control datepicker", required: true,
+              value: @project.start_date.nil? ? "" : @project.start_date.strftime(t "date.rb_format") %>
```
Comment (nhunght) [inline app/views/projects/_setting_information.html.erb]: indent

**Comment by dieunb** [comment]
test this please

File: app/controllers/projects_controller.rb [hunk 2]
```diff
@@ -13,7 +13,24 @@ def show
     end
   end
 
+  def edit
+  end
+
+  def update
+    if @project.update_attributes project_params
+      flash[:success] = flash_message "updated"
```
Comment (nhunght) [inline app/controllers/projects_controller.rb]: flash_message :updated

File: app/controllers/projects_controller.rb [hunk 3]
```diff
@@ -13,7 +13,24 @@ def show
...
+      redirect_to project_path(@project)
+    else
+      flash.now[:failed] = flash_message "not_updated"
```
Comment (nhunght) [inline app/controllers/projects_controller.rb]: flash_message :not_updated

**Pull request merged**